qa/L0_openai/test.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+### Helpers ###
+
+function install_deps() {
+    pushd openai/docker
+    pip install /opt/tritonserver/python/triton*.whl
+    pip install -r requirements.txt
+    if [ "${IMAGE_KIND}" == "TRTLLM" ]; then
+        prepare_tensorrtllm
+    else
+        prepare_vllm
+    fi
+    popd
+}
+
+function prepare_vllm() {
+    echo "No prep needed for vllm currently"
+}
+
+function prepare_tensorrtllm() {
+    MODEL="llama-3-8b-instruct"
+    MODEL_REPO="../tests/tensorrtllm_models"
+    rm -rf ${MODEL_REPO}
+
+    # FIXME: This will require an upgrade each release to match the TRT-LLM version
+    # Use Triton CLI to prepare model repository for testing
+    pip install git+https://github.com/triton-inference-server/triton_cli.git@0.0.10
+    # NOTE: Could use ENGINE_DEST_PATH set to NFS mount for pre-built engines in future
+    triton import \
+        --model ${MODEL}  \
+        --backend tensorrtllm \
+        --model-repository "${MODEL_REPO}"
```

Comment:
Is this going to rebuild the model every time? Do we have a model store for TRT-LLM model? Can be passed to this test script as cached location (with TRT-LLM version generated from) and only generate if TRT-LLM version is different from the cached model

Reply:
I tried to use NFS mount to cache and re-use engines across test jobs, but in appeared that NFS I/O was slower than just building the model each time for now due to some strange slowness going on with infra currently, see example change I had prior: https://github.com/triton-inference-server/server/commit/81ef479164f6d2f5e2fb1c131312fa991f499b95

Ideally this would not build the model each time in the long term.